code/script.js

```diff
@@ -1,325 +1,539 @@
+// Steps to folow
+// 1. Display the game - draw game board  DONE
+// 2. Set secret character by using Math.random() DONE
+// 3. Get the value from selected question DONE
+// 4. Check if the value is matched with the secret person
+//   --> Yes: filter the characters and display only the one matched
+//   --> No: Remove the ones which aren´t matched
+// 5. Build function for Guess button  DONE
+
+
+
 // All the DOM selectors stored as short variables
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
-const restartButton = document.getElementById('restart')
+const restartButton = document.getElementById('restart');
+const findOutButton = document.getElementById("filter");
+const winOrLoseWrapper = document.querySelector(".win-or-lose-wrapper");
+const playAgainBtn = document.getElementById("playAgain");
+const questionSection = document.querySelector(".question-section")
+const winLose = document.querySelector(".secret-person");
+const winOrLoseText = document.getElementById("winOrLoseText");
+const inputWrapper = document.querySelector(".input-wrapper");
+const guessMade = document.querySelector(".guess-made");
+const audioBtn = document.querySelector(".audio-btn");
+const music = document.querySelector(".music");
+
+
+
+// Timing
+const minute = document.querySelector(".minute");
+const second = document.querySelector(".second");
+
+// Sound effect for win or loose section
```

Comment:
Very nice sound effects! They really add to the experience, especially the flipping sounds makes the game more immersive. Maybe add a trombone/dissappointed sound for losing so it doesn't sound the same as when you´re winning?